cmd/template/bluegreen.go

```diff
@@ -122,6 +111,31 @@ strategies:
       shutDownOldVersionAfter:
         - pause:
             untilApproved: true
+
+trafficManagement:
+
+   # Determines the targets that this configuration should be applied to.
+   # If omitted, this configuration will be applied to all targets.
+ - targets: ["<target>"]
+
+   # The Kubernetes traffic management block defines the Kubernetes Service
```

Comment:
We should probably mention that Kubernetes traffic management is required for Blue/Green deployment.